samples/duration_instead_of_dtend.ics

```diff
@@ -0,0 +1,39 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+X-WR-CALNAME:calmozilla1@gmail.com
```

Comment:
Is this one of your addresses? If not, please use something at example.com to avoid adding real addresses in the source.